docs/javascript/v2/how--to-guides/set-up-video-conferencing/render-video/overview.mdx

```diff
@@ -151,3 +151,15 @@ function Conference() {
 ```
 
 </Tab>
+
+
+## Mirror video track
```

Comment:
should be just mirror video. only rendering is mirrored

Reply:
good point, fixing